larray_editor/arraywidget.py

```diff
@@ -536,11 +648,81 @@ class ArrayEditorWidget(QWidget):
     def __init__(self, parent, data, readonly=False, bg_value=None, bg_gradient=None, minvalue=None, maxvalue=None):
         QWidget.__init__(self, parent)
         readonly = np.isscalar(data)
-        self.model = ArrayModel(data, readonly=readonly, parent=self,
-                                bg_value=bg_value, bg_gradient=bg_gradient,
-                                minvalue=minvalue, maxvalue=maxvalue)
-        self.view = ArrayView(self, self.model, data.dtype, data.shape)
-
+        self.readonly = readonly
+
+        self.model_axes = LabelsArrayModel(parent=self, readonly=readonly)
+        self.view_axes = LabelsView(parent=self, model=self.model_axes, position=(LEFT, TOP))
```

Comment:
It is more usual to speak about TOP, LEFT than LEFT, TOP. If that is not driven by underlying requirements, I would change that.